animations/index.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: page
+title: Animations in Flutter
+permalink: /animations/
+---
+
+* TOC Placeholder
+{:toc}
+
+Well designed animations makes a UI feel more intuitive,
+contribute to the slick look and feel of a polished app,
+and improve the user experience.
+Flutter supports several kinds of animations and,
+if you can't find what you want, you can always build your own.
+
+## Animation types
+
+Animations fall into several categories. The following list describes
+several types of animation, and directs you to where you can learn more.
+
+### Animated list or grid
+An animated list (or grid) animates adding, or deleting,
+elements from a list (or grid).
+
+* [AnimatedList example](/catalog/samples/animated-list/)<br>
+This demo, from the [Sample App Catalog](/catalog/samples), shows how to
+animate adding an element to a list, or removing a selected element.
+The internal Dart list is synced as the user modifies the list using
+the plus (+) and minus (-) buttons.
+
+### Hero animation
+
+Also called a _shared element transition_. A common pattern involves
+a grouping of elements, often images. The user clicks one of the images
+and it flies, while growing, and lands (for example) on a new page
+featuring the larger image and more detail.
+Hero animations sometimes occur within the same page.
+
+* [Flutter Gallery](https://github.com/flutter/flutter/tree/master/examples/flutter_gallery)<br>
+You can build the Gallery app yourself, or download it from the Play Store.
+The [Shrine](https://github.com/flutter/flutter/blob/master/examples/flutter_gallery/lib/demo/shrine_demo.dart)
+demo includes an example of a hero animation.
+{% comment %}
+According to Hans, the demo in the Flutter Gallery that is called
+"Animation" isn't really an animation demo. It's more of a
+"scroll driven layout demo".
+{% endcomment %}
+
+### Physics-based animation
+
+In physics-based animation, motion is modeled to resemble real-world
+behavior. When you toss a ball, for example, where and when
+it lands depends on how fast it was tossed, how heavy it is, and how
+far off the ground. Similarly, dropping a ball attached to a spring falls
+(and bounces) differently than dropping a ball attached to a string.
```

Comment:
Should we point advanced users to [AnimationController.animateWith](https://docs.flutter.io/flutter/animation/AnimationController/animateWith.html) and [SpringSimulation](https://docs.flutter.io/flutter/physics/SpringSimulation-class.html)? We can acknowledge that we're working on docs for physics-based animations.